opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -218,7 +242,9 @@ def test_batch_span_processor_lossless(self):
 
         for _ in range(512):
             _create_start_and_end_span("foo", span_processor)
+        from time import sleep  # pylint: disable=import-outside-toplevel
```

Comment:
Nit: I did not mean to add this here, exactly, sorry for the confusion :sweat_smile: This can be added with the rest of the imports :+1: 

Besides that, LGTM!